config/routes.rb

```diff
@@ -46,7 +46,8 @@
   get "/profile", to: "users#show"
   get "/profile/edit", to: "users#edit"
   patch "/profile/edit", to: "users#update"
-
+  get "/profile/edit/password", to: "users#edit_password"
```

Comment:
I wonder if we should consider having a PasswordsController?

Something like the update password in this [article](https://www.sitepoint.com/handle-password-and-email-changes-in-your-rails-api/).  Maybe something to consider for a future refactor!

Reply:
I think this is a good idea, to stay RESTful.